src/components/TeamMembersAddNewMemberModal.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react';
 
-const handleClick = (createUser, handleHideModal) => (event) => {
+// func here takes the hotel on state and checks satff count against plan/team-members
+
+const checkAddEligibility = (plan, staffAmount) => {
+  if (plan === 'free' && staffAmount === 5) {
```

Comment:
Dope! We gotta do this on the backend also

Reply:
It's done! 2 lines of defence 🛡